nomad/local/grapl-local-infra.nomad

```diff
@@ -31,9 +31,8 @@ locals {
       port = 5432,
     },
     {
-      name       = "plugin-work-queue-db",
-      port       = 5433,
-      memory_max = 1024,
```

Comment:
probably worth keeping the `memory_max` here unless there's a compelling reason to remove it

Reply:
It's more like, the only compelling reason it was there in the first place was to get around the OOMs for PWQ. By removing this memory_max= we help validate that this was, indeed, the underlying issue.

Reply:
oh, I see, it "defaults" to a non-default here: https://github.com/grapl-security/grapl/blob/main/nomad/local/grapl-local-infra.nomad#L36

so leaving this unset will in fact _not_ default to 300MB it'll get set to 512MB

>>_<<